### PR TITLE
UC-75: Create Jenkins-based CI pipeline for training an Azure ML model 

### PR DESCRIPTION
**Pull request by sundar-swetha**
### Summary:
- Create a CI pipeline in Jenkins publishing Azure ML pipeline and training a model in Azure ML
- Use docker image as an agent in the Jenkins pipeline
- Load all environment variables from a properties file in Jenkins pipeline

### Test:
Jenkins Pipeline Run: http://merlion-jenkins-vm.eastasia.cloudapp.azure.com:8080/job/swsundar-build-aml-pipeline/

### CODE REVIEW

**Review by daxsorbito — APPROVED** [review]
LGTM 👍

File: jenkins-pipelines/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        docker {
+            //Azure CLI commands require root privileges
+            image 'mcr.microsoft.com/mlops/python:latest'
+            args "--user root --privileged"
+        }
+    }
+    stages {
+        stage('Build, Publish and Run AML Pipeline') {
+            environment {
+                WORKSPACE_NAME = 'swsundar-ml'
```
Comment (ivaramme) [inline jenkins-pipelines/Jenkinsfile]: @sundar-swetha  - if it isn't a lot of work (and I believe it isn't) I think it is better to set these as global variables in the job (and include these values default values) and reference them from the pipeline, keep it clean.

File: jenkins-pipelines/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    }
+                    echo "Environment variables loaded"
+
+                    //Build the Azure ML Pipeline
+                    echo "====== Publish the Azure ML Pipeline ======"
+                    sh 'python -m ml_service.pipelines.diabetes_regression_build_train_pipeline'
+
+                    //Run the Azure ML Pipeline and save the pipeline run id in a text file
+                    echo "====== Run the Azure ML Pipeline ======\n"
+                    sh 'python -m ml_service.pipelines.run_train_pipeline --output_pipeline_id_file "pipeline_id.txt"'
```
Comment (ivaramme) [inline jenkins-pipelines/Jenkinsfile]: Clarification: The plan is to create a new pipeline and run it every time, right? No plans on reusing pipelines, correct?

**Review by ivaramme — CHANGES_REQUESTED** [review]

File: jenkins-pipelines/Jenkinsfile (hunk 1 above)
Comment (sundar-swetha) [reply jenkins-pipelines/Jenkinsfile]: Done.

File: jenkins-pipelines/Jenkinsfile (hunk 2 above)
Comment (sundar-swetha) [reply jenkins-pipelines/Jenkinsfile]: Yes, that's right; for the current usecase we will not be reusing these pipelines.

**Review by ivaramme — APPROVED** [review]

**Pull request merged**